http-clients/src/main/java/com/palantir/remoting/http/errors/SerializableErrorErrorDecoder.java

```diff
@@ -115,6 +118,25 @@ private static Exception constructException(
         // Note: If another constructor is added, then we should refactor the construction logic in order to avoid
```

Comment:
That comment seems to be about the nested try-catch and a worry about nesting further. My change doesn't touch any of that, so I don't think it makes that situation worse. It may be possible to clean that block up, but I think that's a separate issue.